src/style.css

```diff
@@ -0,0 +1,104 @@
+:root {
+  --big-input-height: 70px;
+  --small-input-height: 42px;
+  --big-placeholder-padding: 62px;
+  --small-placeholder-padding: 33px;
+  --big-input-fz: 16px;
+  --small-input-fz: 14px;
+  --big-btn-heiht: 19px;
+  --big-btn-width: 19px;
+  --small-btn-height: 11px;
+  --small-btn-width: 11px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf"),
+    url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  margin: 0 8px;
+}
+
+.search__bar {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+
```

Comment:
Do not leave extra empty lines.

Reply:
I don't knew why these lines are empty, I didn't change them. If I leave them empty before commit, linter will not allow me to make a commit and will force me to fix it.